src/mapgen/mapgen_fractal.cpp

```diff
@@ -66,13 +66,17 @@ MapgenFractal::MapgenFractal(MapgenFractalParams *params, EmergeManager *emerge)
 	julia_z          = params->julia_z;
 	julia_w          = params->julia_w;
 
-	//// 2D terrain noise
-	noise_seabed       = new Noise(&params->np_seabed, seed, csize.X, csize.Z);
+	//// 2D noise
+	if (spflags & MGFRACTAL_TERRAIN)
+		noise_seabed   = new Noise(&params->np_seabed, seed, csize.X, csize.Z);
```

Comment:
Space alignment is now pointless. Please trim the spacing.

Reply:
Was aligned to a line below, but i agree not necesssary. Done.